src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/AccountingXmlDocumentListWrapperFixture.java

```diff
@@ -86,7 +86,9 @@
             "02/26/2018", "xyz789@cornell.edu", "Example multi-doc-type XML file",
             documents(
                     AccountingXmlDocumentEntryFixture.MULTI_DOC_TYPE_TEST_DI,
-                    AccountingXmlDocumentEntryFixture.MULTI_DOC_TYPE_TEST_IB)),
+                    AccountingXmlDocumentEntryFixture.MULTI_DOC_TYPE_TEST_IB,
+                    AccountingXmlDocumentEntryFixture.MULTI_DOC_TYPE_TF1,
+                    AccountingXmlDocumentEntryFixture.MULTI_DOC_TYPE_TF2)),

```

Comment:
Should these enum names be refactored so that it has TEST_ before TF1 and TF2 (MULTI_DOC_TYPE_TEST_TF1) to be consistent with the enum naming conventions in the file?

Reply:
yes it should, I shall change it